src/internal/utils/bucketHelper.ts

```diff
@@ -250,6 +252,10 @@ function bucketSupportsSubtitle(visualizationType: string, bucketLocalIdentifier
         );
     }
 
+    if (visualizationType === VisualizationTypes.BULLET) {
```

Comment:
I don't know about this. it will cause same shit storm like in getChartOptions

Reply:
We agreed to leave it as it is, am I right?